src/help/guides/getting-started.md

```diff
@@ -6,22 +6,23 @@ lang: en-US
 # Getting started
 
 ## Installation
-**Paperback** is currently _not_ on **TestFlight** and _requires_ the usage of **AltStore** in order to install onto your phone. In the future, TestFlight will be used when the lead developer obtains an [Apple Developer Account](https://developer.apple.com/programs/).
+**Paperback** is currently _not_ on **TestFlight** and _requires_ the usage of **AltStore** in order to install onto your iPhone or iPad. In the future, TestFlight will be used when the lead developer obtains an [Apple Developer Account](https://developer.apple.com/programs/).
 
 ::: tip Video tutorial
- If you prefer using a video tutorial, Artesians made a beautiful one. You will find it [here for Windows](https://www.youtube.com/watch?v=n1KRwsxNiWY).
+If you prefer using a video tutorial, Artesians made a beautiful one. You will find it [here](https://www.youtube.com/watch?v=n1KRwsxNiWY) for Windows and [here](https://www.youtube.com/watch?v=CjPjsF4yJ0M) for macOS.
 :::
 
 ### Prerequisites
-1. On your phone, download the app <Download text="from here"/>. The app will download as **`Paperback.ipa`**.
+1. On your iPhone or iPad, download the app <Download text="from here"/>. The app will download as **`Paperback.ipa`**.
 2. Download [AltServer](https://altstore.io/) for your operating system.
 
 ::: tip Pre-releases
- If you're interested in the features introduced in the **Pre-release** or **Stable** builds then you need to be a Patron to use the app.
+On [GitHub](https://github.com/Paperback-iOS/app/releases), there are also pre-releases. These are only available for Patrons. If you are not a Patron, make sure to install the **Stable** builds.
+If you are interested in teasting new features early, consider becoming a Patron.
```

Comment:
Looking for `teasting` on the internet let me think it was probably the intended meaning.
I would suggest:
```suggestion
If you are interested in testing new features early, consider becoming a Patron.
```